Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.7
+
+RUN mkdir -p /app
+EXPOSE 3000
+
+COPY . /app
+WORKDIR /app
+RUN npm install
+RUN npm run build
+
+ENV HOST 0.0.0.0
```

Comment:
What is this line for?

Reply:
Binding to 0.0.0.0 means accepting connections from anywhere.